sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -336,6 +339,9 @@
     for (final String tracingOrigin : propertiesProvider.getList("tracing-origins")) {
       options.addTracingOrigin(tracingOrigin);
     }
+    for (final String proguardUuids : propertiesProvider.getList("proguard-uuids")) {
+      options.addProguardUuid(proguardUuids);
+    }
```

Comment:
let's also add support for Android via `ManifestMetadataReader`

Reply:
Done